<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bankLoginWidget</class>
 <widget class="QWidget" name="bankLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>590</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>银行管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#bankLoginWidget{border-image:url(:/newImage/1.jpg);}</string>
  </property>
  <widget class="QPushButton" name="adminBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>管理员模式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accLineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>441</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pasLineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>439</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pasLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="accLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;账号&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="loginTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>841</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>银行管理系统</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="regBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fogBtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>忘记密码？</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../src/image.qrc"/>
 </resources>
 <connections/>
</ui>
